src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,12 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+### Links without underlines
+
+Links should be underlined by default. Only remove underlines when the number or placement of links makes them difficult to scan.
+
+If you remove underlines on a set of links, there must be clear indication that a user can interact with them. This could be from their content, their placement on the page, or because they are part of a bigger interactive element. For example, navigation links in a header are a widely recognised pattern and are visually different enough to body text to clearly be links.
```

Comment:
We need to make sure we don't conflict with the WCAG advice here:

https://www.w3.org/TR/WCAG20-TECHS/F73.html

> While some links may be visually evident from page design and context, such as navigational links, links within text are often visually understood only from their own display attributes. Removing the underline and leaving only the color difference for such links would be a failure because there would be no other visual indication (besides color) that it is a link.

I *think* this is in alignment, but worth a think just in case.

Reply:
Yeah - we could even reuse the "visually evident from page design and context" phrasing so we can reference W3C if we need to

Reply:
The stuff I wrote here was an attempt to flesh out what the W3C thing means a bit...  deffo don't want to contradict it

Reply:
Just for context, this w3c stuff is guidance and is not used as a basis to enforce WCAG. only the success criteria wording is.

So take it with a big pinch of salt, but is good to align regardless.